### PR TITLE
When 'Block all unencrypted requests' is checked, FTP is still allowed.

### PR DESCRIPTION
**Pull request by Hainish**
This is a fix for Chromium. Firefox still doesn't recognize the
<all_urls> special string for FTP urls.

### CODE REVIEW

**Comment by Hainish** [comment]
@bcyphers care to look this over?

**Comment by bcyphers** [comment]
Sure! I'll take a look later this week.

**Comment by Bisaloo** [comment]
Related: https://github.com/EFForg/https-everywhere/issues/2311

File: chromium/background-scripts/background.js [hunk 1]
```diff
@@ -636,7 +642,7 @@ function onHeadersReceived(details) {
 
 // Registers the handler for requests
 // See: https://github.com/EFForg/https-everywhere/issues/10039
-chrome.webRequest.onBeforeRequest.addListener(onBeforeRequest, {urls: ["*://*/*"]}, ["blocking"]);
+chrome.webRequest.onBeforeRequest.addListener(onBeforeRequest, {urls: ["<all_urls>"]}, ["blocking"]);
```
Comment (bcyphers) [inline chromium/background-scripts/background.js]: Is there a reason to use `<all_urls>` rather than enumerate `["*://*/*", "ftp://*/*"]`?

Comment (Hainish) [reply chromium/background-scripts/background.js]: Firefox doesn't seem to care either way, and never triggers this hook even with FTP urls.  I think you're right though, and enumerating both is a more constrained criteria than just `<all_urls>`.

**Pull request merged**